integrations/upgrade/index.test.ts

```diff
@@ -1000,7 +995,8 @@ test(
       /* Inject missing @config due to nested imports with tailwind imports */
       @import './root.4/base.css';
       @import './root.4/utilities.css';
-      @config '../tailwind.config.ts';
+      @source './**/*.{html,js}';
+      @source './**/*.{html,js}';
```

Comment:
@RobinMalfait This is interesting! We need some bookkeeping I think to avoid adding the config to the same file twice!

Reply:
Fixed it in the next commit (will push in a minute)